Update SSO instructions for new options (#7481)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-microsoft-entra-id.md

<Snippet path="login_url_note" />

### Additional configuration options

> 

Reviewer: Can we open with a line that describes where these config options are located?

Author: These are in the same section as what's covered in "dbt Setup".  I didn't add them directly under there because these are rarely used values (we have low single digit customers using all of these new options combined), but I'm starting to think this section should be moved down there as a sub-section of that heading.  Let me know what you think, @matthewshaver.  (We'll also need to update the screenshot.)

Reviewer: Thanks @asmclean We like to avoid going directly from a header to bullets with no lead in. This is very helpful!

> **Include all groups** will retrieve all groups to which a user belongs from your identity provider.  For example, if Entra ID is configured with nested groups, Entra ID will supply groups to which the user directly belongs and groups under which those groups are nested.  When this option is disabled, only groups where the user has direct membership will be supplied.  This option is enabled by default.

Reviewer: ```suggestion
- **Include all groups:** Retrieve all groups to which a user belongs from your identity provider. If a user is a member of nested groups, it will also include the parent groups. When this option is disabled, only groups where the user has direct membership will be supplied.  This option is enabled by default.
```

> - **Maximum number of groups to retrieve:** Provides a configurable limit to the number of groups to retrieve for users.  By default, this is set to 250 groups, but this number can be increased if users' group memberships exceed that amount.

Reviewer: What happens if the number of groups is set too low? How are the groups pulled into dbt decided? Is it a random sampling or based on some hierarchy?

Author: If the number of groups is set too low, then Entra will likely return an error that results in users seeing an error message displaying "Server error" in an error dialog box on their login page after the attempted login.

> **Maximum number of groups to retrieve** provides a configurable limit to the number of groups to retrieve for users.  By default, this is set to 250 groups, but this number can be increased if users' groups memberships exceed that amount.

Reviewer: ```suggestion
- **Maximum number of groups to retrieve:** Provides a configurable limit to the number of groups to retrieve for users.  By default, this is set to 250 groups, but this number can be increased if users' group memberships exceed that amount.
```
